tgui/packages/tgui/interfaces/GreyscaleModifyMenu.tsx

```diff
@@ -133,16 +131,17 @@ const SingleDirection = (props, context) => {
   const { dir } = props;
   const { data, act } = useBackend<GreyscaleMenuData>(context);
   return (
-    <Table.Cell width="33%">
+    <Flex.Item width="33%">
```

Comment:
Use `grow={1} basis={0}` instead of `width="33%"`, since percentage width defeats the whole purpose of flex.

Reply:
I was able to just remove the properties entirely instead as these were left over when I was first working out the details.